pilot/cmd/pilot-discovery/main.go

```diff
@@ -152,6 +184,28 @@ func init() {
 	discoveryCmd.PersistentFlags().BoolVar(&serverArgs.DiscoveryOptions.EnableCaching, "discoveryCache", true,
 		"Enable caching discovery service responses")
 
+	// Chiron options
+	var chironServiceNames string
+	var chironServiceNamespaces string
+	discoveryCmd.PersistentFlags().BoolVar(&serverArgs.Chiron.EnableChiron, "enableChiron", false,
+		"Enable Chiron")
+	// Specifies the file path to k8s CA certificate.
+	// The default value is configured based on https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/:
+	// the CA certificate bundle is automatically mounted into pods using the default
+	// service account at the path /var/run/secrets/kubernetes.io/serviceaccount/ca.crt.
+	discoveryCmd.PersistentFlags().StringVar(&serverArgs.Chiron.K8sCaCertFile, "k8sCaCertFile", "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
+		"Specifies the file path to k8s CA certificate.")
+	discoveryCmd.PersistentFlags().StringVar(&chironServiceNames, "serviceNames", "istio-galley,istio-sidecar-injector",
+		"The names of the services for which Chiron manage certs.")
+	discoveryCmd.PersistentFlags().StringVar(&chironServiceNamespaces, "serviceNameSpaces", "istio-system,istio-system",
+		"The namespaces of the services for which Chiron manage certs; must be corresponding to the serviceNames parameter.")
+	// Certificate issuance configuration.
+	discoveryCmd.PersistentFlags().Float32Var(&serverArgs.Chiron.CertGracePeriodRatio, "certGracePeriodRatio",
```

Comment:
I would rather keep them as constants - the important one is renewal interval, and that is missing.

Reply:
Done.